src/sentry_unix_pageallocator.c

```diff
@@ -60,7 +60,7 @@ get_pages(size_t num_pages)
 
 #if defined(__has_feature)
 #    if __has_feature(memory_sanitizer)
-    __msan_unpoison(a, page_size_ * num_pages);
+    __msan_unpoison(rv, g_alloc->page_size * num_pages);
```

Comment:
I _think_ this was the intent of this code. The variables just seemed to have been renamed.

Reply:
It was changed by @blinkov in 
https://github.com/getsentry/sentry-native/commit/5dd159f64094c412b99be654d95c3cbabd769090